CRM/Contact/CustomSearchTask.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ */
+
+/**
+ * Class to represent the actions that can be performed on a group of contacts used by the search forms.
+ */
+class CRM_Contact_CustomSearchTask extends CRM_Core_Task {
+
+  /**
+   * @var string
+   */
+  public static $objectType = 'contact';
```

Comment:
I think this should be `'Contact'` per other comment.

Reply:
I was going to implement this but looking at `CRM_Contact_Task` I also see a lower case entity name. 
See: https://github.com/civicrm/civicrm-core/blob/master/CRM/Contact/Task.php#L42

@colemanw if we are going to use capitalized entity names we also need to solve `CRM_Contact_Task` however I do think it is behind the scope of this PR. I am happy to do a follow up PR to fix this after this one is merged. What do you think?